src/App.tsx

```diff
@@ -1,109 +1,37 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
+import { User } from './types/User';
+
+const getComments = (id: number): Comment[] => {
+  const foundComment = commentsFromServer.filter(
+    (comment) => comment.postId === id,
+  );
+
+  return foundComment;
+};
+
+const getUser = (userId: number): User | null => {
```

Comment:
check point 5 in checklist [checklist](https://github.com/mate-academy/react_static-list-of-posts/blob/master/checklist.md)